src/wp-admin/includes/class-wp-comments-list-table.php

```diff
@@ -165,8 +165,6 @@ public function prepare_items() {
 		$_comments = get_comments( $args );
 
 		if ( is_array( $_comments ) ) {
-			update_comment_cache( $_comments );
-
```

Comment:
This change doesn't seem directly related to this PR, am I missing something? 

I get that they should be cached by the `get_comments()` call.

Reply:
It is related. `update_comment_cache` calls `update_meta_cache`, which primes comment meta. This is not needed, as get_comment / WP_Comment_Query does this all for you. This line is a hang over from before `WP_Comment_Query` and is safe to remove now. I reviewed all places that used get_comment / WP_Comment_Query, which is how I found this.